governance/remote_executor/cli/src/main.rs

```diff
@@ -87,9 +87,11 @@ fn main() -> Result<()> {
             let wormhole_config_data =
                 Config::try_from_slice(&rpc_client.get_account_data(&wormhole_config)?)?;
 
+            let guardian_set_data_offset = if cli.chain == 26 { 0 } else { 8 }; // Pythnet's guardian set account has no discriminator
```

Comment:
Pythnet's wormhole is pre-wormhole rewrite so accounts don't have 8 bytes discriminators at the beginning.
On the other hand our wormhole receiver that we will post to every svm, does.